onnx/version_converter/IntraDomainConverter.h

```diff
@@ -0,0 +1,207 @@
+// Converter for ONNX models between different opset versions
+// in the same domain.
+
+#pragma once
+
+#include "onnx/version_converter/BaseConverter.h"
+
+namespace ONNX_NAMESPACE { namespace version_conversion {
+
+class IntraDomainVersionConverter : BaseVersionConverter {
+  private:
+    bool DEBUG = false;
+
+  public:
+    IntraDomainVersionConverter() {
+      // TODO: Register adapters to the version converter
```

Comment:
This is not a TODO, just a note :-)

Reply:
Also do we really want to register any adapter directly in intra one? probably just delete this comment.